lib/resources/majors.tsx

```diff
@@ -2425,144 +2479,238 @@ const bsCS_New: Major = {
       required_credits: 40,
       min_credits_per_course: 1,
       description:
-        "For more information please visit the <a href='https://www.cs.jhu.edu/2021undergraduate-advising-manual/'>" +
+        "For more information please visit the <a href='https://www.cs.jhu.edu/undergraduate-studies/academics/ugrad-advising-manual/'>" +
         'major degree requirement</a> section on the department website.',
-      criteria:
-        'EN Computer Science[D]^OR^CSCI-OTHER[T]^OR^Gateway Computing[N]',
+      criteria: 'EN Computer Science[D]^OR^EN.500.112[C]^OR^EN.660.400[C]',
+      double_count: [
+        'Mathematics',
+        'Humanities/Social Sciences',
+        'Writing Intensive',
+      ],
```

Comment:
shouldn't it double count with Computer Science Classification also?

Reply:
You're right. Done